scylla/src/transport/connection.rs

```diff
@@ -1359,24 +1360,41 @@ impl Connection {
         };
 
         let query_response = self.query_raw_unpaged(&query, PagingState::start()).await?;
+        self.handle_use_keyspace_result(keyspace_name, query_response)
+    }
 
+    fn handle_use_keyspace_result(
+        &self,
+        keyspace_name: &VerifiedKeyspaceName,
+        query_response: QueryResponse,
+    ) -> Result<(), QueryError> {
```

Comment:
> Another think: in recent force push you changed this method from pub(crate) to pub? Why?

Wojciech suggested to remove it: https://github.com/scylladb/scylla-rust-driver/pull/1080#discussion_r1777223293

Reply:
From what I see, it's changed from `pub(crate)` to `pub(self)` (implicit, because no visibility specifier is written).

Reply:
> I'm not sure this is the best name for this method. It doesn't really handle anything - maybe `check_use_keyspace_result`?
> 
> Also it doesn't use self in the body - it can be a free function.

Removed `&self` parameter and renamed to `verify_use_keyspace_result`.

Reply:
> From what I see, it's changed from `pub(crate)` to `pub(self)` (implicit, because no visibility specifier is written), not `pub` as @Lorak-mmk stated.

Oh, I misread the comment. That's right, we decreased the visibility